### DISCUSSION TITLE
Flattened structure in target folder

### DISCUSSION THREAD

**Discussion by XeNz** [parcel-bundler/parcel#7324] [Q&A] [answered]
Given I have the following sources structure:

📦src
 ┣ 📂components
 ┃ ┣ 📜some.component.tsx
 ┣ 📂pages
 ┃ ┣ 📂some-specific-page
 ┃ ┃ ┗ 📜some-specific-page.tsx
 ┃ ┣ 📜main.ts

And the following sources configuration: 
```
  "source": [
    "src/pages/main.ts",
    "src/components/some.component.tsx",
    "src/pages/some-specific-page/some-specific-page.tsx"
  ],
```
How would I need to configure parcel 2 to get a flattened structure such as below in my target folder?

📦target
 ┣ 📜main.js
 ┣ 📜some.component.js
 ┣ 📜some-specific-page.js

**Comment by gerardnico** [comment, marked as answer]
If you use several target (one by source), you can specify the output for each. [multiple-targets](https://parceljs.org/features/targets/#multiple-targets)

```json
{
  "targetname":"./dist/targetname.js"
}
```
Not really sure if this is the path to go

**Reply by XeNz** [reply]
Looks like I missed this in the documentation. It's a pity it does create some duplication, but it will do for now. Thanks!

**Comment by gerardnico** [comment]
Also, normally, you would only give the `src/pages/main.ts` as source no ? You would get a bundle

**Reply by XeNz** [reply]
I currently have a use-case where I need a main bundle but also need a small amount hot pluggable components, hence the splitting into multiple targets, yet wanting to keep it simple so that it's flattened to a single directory.
